runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -61,12 +63,21 @@ private InventoryViewerOverlay(Client client, ItemManager itemManager, Inventory
 		panelComponent.setOrientation(ComponentOrientation.HORIZONTAL);
 		this.itemManager = itemManager;
 		this.client = client;
+		this.config = config;
 		this.hidden = config.hiddenDefault();
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		if (config.hiddenInventory())
+		{
+			if (client.getVar(VarClientInt.INVENTORY_TAB) == 3)
```

Comment:
Store the current inventory tab in the plugin and check the plugin's value here to avoid potentially calling `client.getVar()` every frame